### PR TITLE
OWLS-103387 - Add FAQ about the active processor count for burstable pods with no CPU limit.

### PR DESCRIPTION
**Pull request by ankedia**
Add FAQ about the incorrect JVM interpretation of CPU requests causing the CPU underutilization and incorrect number of GC threads.

### CODE REVIEW

File: documentation/4.0/content/faq/resource-settings.md [hunk 1]
```diff
@@ -199,6 +199,13 @@ There's no memory or CPU limit configured by default in samples and so the defau
...
 
+#### Burstable pods and JDK active processor count calculation
+If you have [Burstable Pods](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-burstable) that configure the CPU resource requests but have no CPU limits, then JDK can incorrectly calculate the active processor count. The JDK interprets the value of `--cpu-shares` parameter (which maps to `spec.containers[].resources.requests.cpu`) to limit how many CPUs the current process can use as explained in [JDK-8288367](https://bugs.openjdk.org/browse/JDK-8288367). This might cause the JVM to use fewer CPUs than available, leading to an under utilization of CPU resources when running in Kubernetes environment. This will be fixed when the JDK is updated to newer versions `JDK 8u371` or `JDK 11.0.17`.
```
Comment (rosemarymarano) [inline documentation/4.0/content/faq/resource-settings.md:203]: then JDK -> then the JDK
of `--cpu-shares` parameter -> of the `--cpu-shares` parameter
use as explained in -> use, as explained in
in Kubernetes environment -> in a Kubernetes environment
(Suggested re-write) Updating the JDK to newer versions, `JDK 8u371` or `JDK 11.0.17`, fixes this.

File: documentation/4.0/content/faq/resource-settings.md [hunk 2]
```diff
@@ -199,6 +199,13 @@ There's no memory or CPU limit configured by default in samples and so the defau
...
 
+#### Burstable pods and JDK active processor count calculation
+If you have [Burstable Pods](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-burstable) that configure the CPU resource requests but have no CPU limits, then JDK can incorrectly calculate the active processor count. The JDK interprets the value of `--cpu-shares` parameter (which maps to `spec.containers[].resources.requests.cpu`) to limit how many CPUs the current process can use as explained in [JDK-8288367](https://bugs.openjdk.org/browse/JDK-8288367). This might cause the JVM to use fewer CPUs than available, leading to an under utilization of CPU resources when running in Kubernetes environment. This will be fixed when the JDK is updated to newer versions `JDK 8u371` or `JDK 11.0.17`. 
+
+{{% notice note %}}
+You can use the `-XX:ActiveProcessorCount` java option to override the number of CPUs that the JVM automatically detects and uses when creating threads for various subsystems.
```
Comment (rosemarymarano) [inline documentation/4.0/content/faq/resource-settings.md:206]: (Suggested re-write) To override the number of CPUs that the JVM automatically detects and uses when creating threads for various subsystems, use the `-XX:ActiveProcessorCount` Java option.

**Review by rosemarymarano — COMMENTED** [review]
Few edits and suggestions

**Review by rosemarymarano — APPROVED** [review]
Perfect. Thank you.

File: documentation/4.0/content/faq/resource-settings.md (hunk 1 above)
Comment (ankedia) [reply documentation/4.0/content/faq/resource-settings.md]: Fixed, thanks.

**Review by rjeberhard — APPROVED** [review]

**Pull request merged**